routes/order.route.js

```diff
@@ -0,0 +1,15 @@
+const orderController = require('../controllers/order.controller');
+const {authJwt} = require('../middlewares/index');
+const {validateOrder} = require('../middlewares')
+
+module.exports = (app) => {
+
+    app.post('/orderService/api/v1/orders', [authJwt.verifyToken, validateOrder.validateOrderRequestBody], orderController.create);
+
+    app.put('/orderService/api/v1/orders/:id', [authJwt.verifyToken, authJwt.isAdminOrOwnerOfOrder], orderController.update);
+
+    app.put('/orderService/api/v1/order/:id', [authJwt.verifyToken, authJwt.isAdminOrOwnerOfOrder], orderController.cancle);
```

Comment:
Combine both of these in a single API .... 

Add the logic in the code to handle the cancellation